python/py-peakutils/Portfile

```diff
@@ -17,13 +17,15 @@ maintainers         {reneeotten @reneeotten} openmaintainer
 description         Peak detection utilities for 1D data
 long_description    ${description}
 
-homepage            https://peakutils.readthedocs.io
+homepage            https://bitbucket.org/lucashnegri/peakutils/
+distfiles           v${version}.tar.gz
+master_sites        https://bitbucket.org/lucashnegri/peakutils/get/
 
-checksums           rmd160  fb665e5e469b9251648c39095c71de02760248b7 \
-                    sha256  390696b7ffc7933ff1426d40fcb3079a79b99fcdf1f64e4122fce82584d77b00 \
-                    size    118684
+checksums           rmd160  0502dde7cba8e2c8056a8e950ed21d399f759166 \
+                    sha256  5bb4fd0ca9289158d56af53658f258b1006ce61d499534cc47b7c0d365102caa \
+                    size    187589
 
-python.versions     39
+python.versions     27 35 36 37 38 39
```

Comment:
we certainly don't want to add EOL Python versions so `27` and `35` are out of the window ;) If you need the earlier Python versions for anything, we can add them but if not I'd prefer to stick to the latest one only.

Reply:
Right. How about a compromise? We'll go with `37`, `38`, and `39` only. I could be persuaded to drop `37`, but I use `38` on some computers and have to hold the line there.

Reply:
sounds good!